packages/nimble-components/build/generate-workers/index.ts

```diff
@@ -24,25 +24,34 @@ function writeFile(filePath: string, content: string): void {
 
 const renderModuleName: string = '../bundle/matrix-renderer.js';
 const workerTypes: string = '../esm/source/types.d.ts';
+const matrixRendererTypes: string = '../esm/source/matrix-renderer.d.ts';
 const workersDirectory: string = path.resolve('./src/wafer-map/workers');
 
 prepareDirectory(workersDirectory);
 
 const modulePath: string = resolveModulePath(renderModuleName);
 const sourceCode: string = fs.readFileSync(modulePath, 'utf-8');
 const workerTypesPath: string = resolveModulePath(workerTypes);
-const typesCode: string = fs.readFileSync(workerTypesPath, 'utf-8');
+const workerTypesCode: string = fs.readFileSync(workerTypesPath, 'utf-8');
+const matrixRendererTypesPath: string = resolveModulePath(matrixRendererTypes);
+const matrixRendererTypesCode: string = '/* eslint-disable @typescript-eslint/prefer-readonly */\n'
+    + `/* eslint-disable @typescript-eslint/explicit-member-accessibility */\n${fs.readFileSync(
+        matrixRendererTypesPath,
+        'utf-8'
+    )}`;
 
 const fileContent: string = `// eslint-disable-next-line no-template-curly-in-string
 export const workerCode = ${JSON.stringify(sourceCode)};
 `;
```

Comment:
Instead of putting the eslint-disable lines in the generated code can you add src/wafer-map/workers/ to the ignore list: 
https://github.com/ni/nimble/blob/5778a15ee6098377ed5044f014c9b18151313c57/packages/nimble-components/src/.eslintrc.js#L8